cpp/log/leveldb_db-inl.h

```diff
@@ -47,18 +47,37 @@ std::unique_ptr<const leveldb::FilterPolicy> BuildFilterPolicy() {
 #endif
 
 
-std::string FormatSequenceNumber(const int64_t seq) {
-  return std::to_string(seq);
+std::string IndexToKey(int64_t index) {
+  CHECK_EQ(8, sizeof(index));
```

Comment:
Also, this is `index` I'm looking at, I'm more worried someone will change the type of the parameter, which would invalidate existing databases...